src/bigquery.ts

```diff
@@ -2206,7 +2206,7 @@ export class BigQuery extends Service {
     this.runJobsQuery(queryReq, (err, job, res) => {
       this.trace_('[runJobsQuery callback]: ', query, err, job, res);
       if (err) {
-        (callback as SimpleQueryRowsCallback)(err, null, res);
+        (callback as SimpleQueryRowsCallback)(err, null, job);
```

Comment:
Learning Leah - why pass job here now when we passed res before?

Reply:
This is to fix an error that the user also noticed on the mentioned issue. The `SimpleQueryRowsCallback` third parameter should be an `IJob` and before it was returning the `jobs.query` Response here.